### PR TITLE
feat: Add schemas and relations selectors

### PR DESCRIPTION
**Pull request by ClaudioRussi**

### CODE REVIEW

File: dbt_coves/tasks/generate/sources.py [hunk 1]
```diff
@@ -60,13 +67,24 @@ def run(self):
         db = self.config.credentials.database
         schema_names = [schema.upper() for schema in self.get_config_value("schemas")]
 
+        schema_selectors = []
+        for schema_name in schema_names:
+            if "*" in schema_name:
+                schema_selectors.append(schema_name.replace("*", ".*"))
```
Comment (ssassi) [inline dbt_coves/tasks/generate/sources.py:73]: what happens if the schema_name does not contain '*'?

File: dbt_coves/tasks/generate/sources.py [hunk 2]
```diff
@@ -60,13 +67,24 @@ def run(self):
...
                 if schema != "INFORMATION_SCHEMA"
             ]
+
+            for schema in schemas:
+                for selector in schema_selectors:
+                    if re.search(selector, schema):
+                        schema_names.append(schema)
+                        break
+        
             filtered_schemas = list(set(schemas).intersection(schema_names))
```
Comment (ssassi) [inline dbt_coves/tasks/generate/sources.py:88]: this intersection won't work if schema_names items contain '*', don't it?

File: dbt_coves/tasks/generate/sources.py [hunk 3]
```diff
@@ -84,10 +102,24 @@ def run(self):
                     filtered_schemas = selected_schemas
                 else:
                     return 0
+
+            rel_names = [relation.upper() for relation in self.get_config_value("relations")]
+            rel_selectors = []
+            for rel_name in rel_names:
+                if "*" in rel_name:
```
Comment (ssassi) [inline dbt_coves/tasks/generate/sources.py:109]: if '*' not in rel_name, rel_selectors will be empty

**Review by ssassi — CHANGES_REQUESTED** [review]

File: dbt_coves/tasks/generate/sources.py (hunk 1 above)
Comment (ClaudioRussi) [reply dbt_coves/tasks/generate/sources.py:73]: If the schema_name doesn't contain *, it will use the name as it was used before

**Pull request merged**